src/Transformation.php

```diff
@@ -93,7 +93,7 @@ public function createManifestMetadata(
                     $column->getColumnName(),
                     $dataTypes,
                     $column->getColumnDefinition()->isNullable(),
-                    false, // primary key is not supported in keboola/table-backend-utils
```

Comment:
tohle byl kec a nebo to už neplatí, ale `keboola/table-backend-utils` jsem neupdatoval